java/client/src/main/java/glide/api/RedisClusterClient.java

```diff
@@ -118,4 +120,28 @@ public CompletableFuture<ClusterValue<String>> info(
         return commandManager.submitNewCommand(
                 Info, options.toArgs(), route, response -> ClusterValue.of(handleObjectResponse(response)));
     }
+
+    /** {@inheritDoc} The command will be routed automatically to all nodes. */
+    @Override
+    public CompletableFuture<String> configRewrite() {
```

Comment:
Doesn't ClusterClient extend baseClient why do you need to call it like this? We don't have the other commands such as incr etc...

Reply:
I do it to redefine (extend) the javadoc. The function signature (= API) is the same.

Reply:
reverted